### PR TITLE
Feat/d4 g 208 make consumption header sticky

### PR DESCRIPTION
**Pull request by Baboo7**
## Proposed changes

ticket: https://ogre-d4g.atlassian.net/browse/D4G-208

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

---

What types of changes does your code introduce to ogre ?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Other (please describe):

## Checklist

---

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

offseason_ogre/blob/master/CONTRIBUTING.md) doc

- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation and modify the README.md (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] If the feature changes current behavior, reasons why your solution is better.

## What is the current behavior?

---

-
-
-

## What is the new behavior?

---

-
-
-

## Further comments

---

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

## Where has this been tested?

---

Docker version: …
Operating System: …
node version: …
postgresql version: …

image
image

### CODE REVIEW

File: packages/client/src/modules/play/Components/PlayBox.tsx [hunk 1]
```diff
@@ -1,9 +1,52 @@
...
+  ...boxProps
+}: { header?: ReactNode; headerSticky?: boolean } & BoxProps = {}) {
   const theme = useTheme();
+
+  if (header) {
+    return (
+      <>
+        <HeaderSection headerSticky={headerSticky}>
+          <HeaderMask />
+          <Box
```
Comment (remiriv) [inline packages/client/src/modules/play/Components/PlayBox.tsx]: @Baboo7 On pourrait factoriser ici en définissant un style avec ce qui est commun entre cette Box, celle après et celle qui existait déjà avant

Comment (remiriv) [reply packages/client/src/modules/play/Components/PlayBox.tsx]: `const StyledPlayBox = styled(Box)(({ theme }) => ({
      position: "relative",
      bgcolor={theme.palette.primary.main},
      border="2px solid",
      borderColor="white",
      color="white"
}));
`

**Review by remiriv — APPROVED** [review]

File: packages/client/src/modules/play/Components/PlayBox.tsx (hunk 1 above)
Comment (remiriv) [reply packages/client/src/modules/play/Components/PlayBox.tsx]: j'ai raté ma proposition de code mais t'as l'idée haha

**Comment by remiriv** [comment]
@Baboo7 Top ça marche nickel ! Tellement plus lisible avec ça! 
Petite question, pourquoi le favicon est marqué dans les fichiers modifiés?

**Comment by Baboo7** [comment]
@remiriv la favicon était rectangulaire au lieu d'être carrée. Du coup elle était étirée horizontalement dans l'onglet du navigateur

File: packages/client/src/modules/play/Components/PlayBox.tsx (hunk 1 above)
Comment (Baboo7) [reply packages/client/src/modules/play/Components/PlayBox.tsx]: C'est mis à jour 👌

Comment (remiriv) [reply packages/client/src/modules/play/Components/PlayBox.tsx]: Seems good tu peux merge !

**Comment by remiriv** [comment]
@Baboo7 wah bien vu pour la favicon j'avais absolument pas fait gaffe

**Pull request merged**